packages/analytics/tests/index.spec.ts

```diff
@@ -0,0 +1,49 @@
+import Chance from 'chance';
+import CentralizedAnalytics from '../src';
+
+describe('mixpanel', () => {
+  const chanceHelper = new Chance();
+  it('should save user data', async () => {
+    const cpm = new CentralizedAnalytics({ project: chanceHelper.word() });
+    cpm.createAccount({
+      accountId: Buffer.from(chanceHelper.integer().toString()),
+      firstname: chanceHelper.first(),
+      lastname: chanceHelper.last(),
+      email: chanceHelper.email(),
+      organizationId: Buffer.from(chanceHelper.integer().toString()),
+      created: new Date(),
+    });
+  });
```

Comment:
Find a way to spy on Mixpanel library to check whether the methods are called with the expected arguments.